pkg/pillar/cmd/zedmanager/updatestatus.go

```diff
@@ -435,28 +436,31 @@ func doActivate(ctx *zedmanagerContext, uuidStr string,
 		if remaining < need {
 			var errStr string
 			var entities []*types.ErrorEntity
-			errSeverity := types.ErrorSeverityWarning
-			retryCondition := "Retry will be triggered when apps will shutdown: "
+			var domains []string
+			errSeverity := types.ErrorSeverityError
 			if remaining+halting < need {
 				errStr = fmt.Sprintf("Remaining memory bytes %d app instance needs %d",
 					remaining, need)
-				for _, st := range ctx.pubAppInstanceStatus.GetAll() {
-					status := st.(types.AppInstanceStatus)
-					entities = append(entities, &types.ErrorEntity{EntityID: status.UUIDandVersion.UUID.String(), EntityType: types.ErrorEntityAppInstance})
-					retryCondition = fmt.Sprintf("%s %s;", retryCondition, status.DisplayName)
-				}
 			} else {
 				errStr = fmt.Sprintf("App instance needs %d bytes but only have %d; waiting for halting app instances to free up %d bytes",
```

Comment:
Might make sense to s/waiting for/waiting for one or more/

Reply:
Added